gulpfile.js/config.js

```diff
@@ -5,7 +5,13 @@ module.exports = {
   browserSync: {
     ghostMode: false,
     open: false,
-    proxy: host
+    proxy: host,
+    watchOptions: {
+      ignoreInitial: true
+    },
+    files: ['dist/{Components,Features}/**/*.css'],
+    injectChanges: true,
+    reloadDebounce: 1000
```

Comment:
isn‘t this too high?

Reply:
Maybe.. I'll leave it in for now and see how people like it; it's mainly there to fix _variables.styl reloading, but we could probably reduce it a bit if it's too much for other files.